src/containers/Chat/ChatMessages/ChatInput/ChatInput.test.tsx

```diff
@@ -170,7 +179,14 @@ describe('<ChatInput />', () => {
     propsWithBspStatusNone.contactBspStatus = 'NONE';
     const { getByText } = render(
       <MockedProvider mocks={mocks} addTypename={false}>
-        <ChatInput {...propsWithBspStatusNone} />
+        <LexicalComposer
+          initialConfig={{
+            namespace: 'chat-input',
+            onError: (error: any) => console.log(error),
+          }}
+        >
+          <ChatInput {...propsWithBspStatusNone} />
+        </LexicalComposer>
       </MockedProvider>
```

Comment:
This seems to be common in all the test cases. Create a function for this and reuse.
```suggestion
   const chatInput= (props) =>
        <LexicalComposer
          initialConfig={{
            namespace: 'chat-input',
            onError: (error: any) => console.log(error),
          }}
        >
          <ChatInput {...props} />
        </LexicalComposer>
      </MockedProvider>
```